lib/libmbedtls/core/des3_cmac.c

```diff
@@ -0,0 +1,160 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (C) 2018, ARM Limited
+ * Copyright (C) 2019, Linaro Limited
+ * Copyright (C) 2021, SumUp Services GmbH
+ */
+
+#include <assert.h>
+#include <compiler.h>
```

Comment:
Not needed I think

Reply:
Yes, I agree but it was taken from
https://github.com/OP-TEE/optee_os/blob/master/lib/libmbedtls/core/aes_cmac.c#L7-L8
Do you want me to remove it?

Reply:
Of course, otherwise it's called cargo-cult programming and it's a bad thing ;-)

Reply:
Done:)

Reply:
I guess <assert.h> is actually needed as Shippable has just reported build error.

Reply:
I added back <assert.h>.
Now it builds properly.

Reply:
Yes my comment was only about the line immediately above, sorry for not being clear.